python/cac_tripplanner/destinations/views.py

```diff
@@ -130,3 +133,24 @@ def get(self, request, *args, **kwargs):
             results = results[:limit_int]
         data = serializers.serialize('json', results, fields=output_fields)
         return HttpResponse(data, 'application/json')
+
+class FeedEvents(View):
+    """ API endpoint for the FeedEvent model """
+
+    def get(self, request, *args, **kwargs):
+        """ GET 20 most recent feed events that are published
+
+        TODO: Additional filtering, dynamic limits?
```

Comment:
Yes, that was my impression as well, that they'd go into the featured locations and be used for routing.  We'll also want to remove them once event dates pass, I imagine.